### ISSUE TITLE
Got a Tiramisu/.rpx version?

### ISSUE THREAD

**Issue by SwirlyStone5877** [wiiu-env/sdcafiine_plugin#13]
I have my Wii U modded with Tiramisu, and I was wondering... Do you have a .rpx version?

**Comment by Alvarden15** [comment]
Would be cool to have that version, if possible.

**Comment by Alvarden15** [comment]
Yeah, it's fair that it's not planned, at this point it's better to use Aroma than Tiramisu.
